static/js/components/ProgramCourseList.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import React from 'react';
+import ProgramCourse from './ProgramCourse';
+
+
+export default class ProgramCourseList extends React.Component {
+  props: {
+    courses: Array<Object>,
+  }
+  render() {
+    const courseDetails = this.props.courses.map((course, index) =>
+      <ProgramCourse key={index} {...course} />
+    );
```

Comment:
like above, my pref here would be a method that takes all the courses and returns this, which you then call below